ql/src/test/queries/clientpositive/my_sanity_tpcds.q

```diff
@@ -0,0 +1,1241 @@
+add jar /home/msydoron/eclipse-workspace/JethroDataJDBCDriver/target/jethro-jdbc-3.6-standalone.jar;
```

Comment:
We cannot include this file. Could you move some tests (including explain command) to the example you created in ```jdbc_handler.q``` and remove it? The goal would be to have coverage in the tests for the important cases that you expect to be pushed to JDBC, so we can basically control that we do not regress in upcoming Calcite releases, etc.

Same for any q test that relies on Jethro.